### PR TITLE
[MEX-505] Fix token trending score

### PR DESCRIPTION
**Pull request by claudiulataretu**
## Reasoning
- trending score isn't performing right on edge cases
  
## Proposed Changes
- added minimum score to be used in case of edge cases on token metrics changes

## How to test
```
query Tokens {
    filteredTokens(
        pagination: {
            first: 200
        }
        filters: {
            enabledSwaps: false
        }
        sorting: {
            sortField: TRENDING_SCORE
            sortOrder: DESC
        }
    ) {
        edges {
            node {
                identifier
                trendingScore
            }
        }
    }
}
```

### CODE REVIEW

File: src/modules/tokens/services/token.compute.service.ts [hunk 1]
```diff
@@ -232,7 +232,7 @@ export class TokenComputeService implements ITokenComputeService {
             metric: 'priceUSD',
         });
 
-        return values24h[0]?.value ?? undefined;
```
Comment (lcswillems) [inline src/modules/tokens/services/token.compute.service.ts:235]: Should we include this fix? It seems to be unrelated to the issue we were seeing but rather related to handling the case of tokens launched in the last day. Maybe we want to properly address this later?

Comment (claudiulataretu) [reply src/modules/tokens/services/token.compute.service.ts]: done

**Review by lcswillems — APPROVED** [review]

**Review by mad2sm0key — APPROVED** [review]
